src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -122,7 +122,64 @@ In the right part, we see the *graph of dependent tasks* composing the workflow.
 
 Finally, above the workflow graph, we see the various *palettes* which can be used to _drag & drop_ *sample task definitions* or *complex constructs*.
 
-The following chapters describe the three main type of tasks which can be defined inside ProActive Workflows.
+The link:../user/ProActiveUserGuide.html#_types_of_tasks[following chapters] describe the three main type of tasks which can be defined inside ProActive Workflows.
+
+=== Use the ProActive Catalog from the Studio
+
+The GUI interaction with the Catalog can be done in two places: the +++<a class="studioUrl" href="/studio" target="_blank">Studio</a>+++ and +++<a class="automationDashboardUrl" href="/automation-dashboard/#/portal/catalog-portal" target="_blank">Catalog Portal</a>+++.
```

Comment:
why are you using <a> tags here? I cannot render to see the difference between the tag and the regular  link[text^]

Reply:
@luispinedamx the <a> tag is used to directly write html code and especially here to provide a class name for the link tag.

This is used by a custom script to rewite href target links between online and embedded documentation.

Reply:
@luispinedamx I believe tags were added by @MaelAudren at some point in the past in order to make documentation links always link to internal portals (not try portals) in a relative-url fashion.
I didn't add or touch anything in the texts in this PR, I just modified the order.